package.json

```diff
@@ -26,6 +26,7 @@
     "commander": "^2.9.0",
     "gitclone": "^2.0.0",
     "github": "^14.0.0",
+    "github-org-repos": "^1.0.2",
     "github-repositories": "^3.0.0",
```

Comment:
Should the old dependency be dropped?

Reply:
Most likely, yes... Didn't check if it's still used, however.